python/lsst/jointcal/jointcal.py

```diff
@@ -329,8 +340,8 @@ def run(self, dataRefs, profile_jointcal=False):
         return pipeBase.Struct(dataRefs=dataRefs, oldWcsList=oldWcsList, metrics=self.metrics)
 
     def _do_load_refcat_and_fit(self, associations, defaultFilter, center, radius,
-                                name="", refObjLoader=None, fit_function=None, tract=None,
-                                profile_jointcal=False, match_cut=3.0):
+                                name="", refObjLoader=None, filters=[], fit_function=None,
+                                tract=None, profile_jointcal=False, match_cut=3.0):
```

Comment:
I'm not sure which of the python naming standards applies to jointcal: most of the python in it I wrote after we said "you can use _ in python".